src/test/functional/steps/court-addresses.ts

```diff
@@ -0,0 +1,156 @@
+  expect(selectorPrimaryAddressTownExist).equal(true);
+  await I.clearField(selectorPrimaryAddressTown);
+
+  const selectorPrimaryAddressTownWelsh = '#primaryAddressTownWelsh';
+  const selectorPrimaryAddressTownWelshExist = await I.checkElement(selectorPrimaryAddressTownWelsh);
+  expect(selectorPrimaryAddressTownWelshExist).equal(true);
+  await I.clearField(selectorPrimaryAddressTownWelsh);
+
+  const selectorPrimaryAddressPostcode = '#primaryAddressPostcode';
+  const selectorPrimaryAddressPostcodeExist = await I.checkElement(selectorPrimaryAddressPostcode);
+  expect(selectorPrimaryAddressPostcodeExist).equal(true);
+  await I.clearField(selectorPrimaryAddressPostcode);
+});
+
+When('I select the Address Type {string}', async (addressType: string) => {
+  const selector = '#primaryAddressType';
+  const elementExist = await I.checkElement(selector);
+  expect(elementExist).equal(true);
+  await I.selectItem(selector,addressType);
+});
+
+Then('I enter court {string} in the Address textbox', async (address: string) => {
+  const selector = '#primaryAddressLines';
+  const elementExist = await I.checkElement(selector);
+  expect(elementExist).equal(true);
+  await I.setElementValueForInputField(selector,address);
+});
+
+Then('I enter {string} in the Address Welsh textbox', async (welshAddress: string) => {
+  const selector = '#primaryAddressWelsh';
+  const elementExist = await I.checkElement(selector);
+  expect(elementExist).equal(true);
+  await I.setElementValueForInputField(selector,welshAddress);
+});
+
+Then('I enter {string} in the Town textbox', async (town: string) => {
+  const selector = '#primaryAddressTown';
+  const elementExist = await I.checkElement(selector);
+  expect(elementExist).equal(true);
+  await I.setElementValueForInputField(selector,town);
+});
+
+Then('I enter {string} in the town Welsh textbox', async (welshTown: string) => {
+  const selector = '#primaryAddressTownWelsh';
+  const elementExist = await I.checkElement(selector);
+  expect(elementExist).equal(true);
+  await I.setElementValueForInputField(selector,welshTown);
+});
+
+Then('I enter {string} in the postcode textbox', async (postcode: string) => {
+  const selector = '#primaryAddressPostcode';
+  const elementExist = await I.checkElement(selector);
+  expect(elementExist).equal(true);
+  await I.setElementValueForInputField(selector,postcode);
+});
+
+Then('I click the Save Addresses button', async () => {
+  const selector = 'button[name="saveAddresses"]';
+  const elementExist = await I.checkElement(selector);
+  expect(elementExist).equal(true);
+  await I.click(selector);
+});
+
+Then('A green message is displayed for the updated address {string}', async (message: string) => {
+  const selector = '#addressesContent > div.govuk-panel.govuk-panel--confirmation > h1';
+  expect(await I.checkElement(selector)).equal(true);
+  const messageUpdate = await I.getElement('#addressesContent > div.govuk-panel.govuk-panel--confirmation > h1');
```

Comment:
Can use the selector variable in the following: 

const messageUpdate = await I.getElement('#addressesContent > div.govuk-panel.govuk-panel--confirmation > h1');

to 

const messageUpdate = await I.getElement(selector);